source/fuzz/fuzzer_util.h

```diff
@@ -93,7 +93,9 @@ bool CanInsertOpcodeBeforeInstruction(
     SpvOp opcode, const opt::BasicBlock::iterator& instruction_in_block);
 
 // Determines whether it is OK to make a synonym of |inst|.
-bool CanMakeSynonymOf(opt::IRContext* ir_context, opt::Instruction* inst);
+bool CanMakeSynonymOf(opt::IRContext* ir_context,
```

Comment:
Can you update the comment to remark that - among other things - we cannot make a synonym of an id that participates in an irrelevant id fact?

Reply:
Fixed in 35f5730.